src/codeview.js

```diff
@@ -232,6 +239,7 @@ var Codeview = GObject.registerClass({
             });
             this._cached_ast = null;
             this.compile(false);
+            this.notify('text');
```

Comment:
The notify is automatic at least in python, unless you add the`GObject.ParamFlags.EXPLICIT_NOTIFY` flag to the property. Isn't the same in GJS?

Reply:
It's automatic if you use [`GObject.set_property()`](https://gjs-docs.gnome.org/gobject20~2.0_api/gobject.object#method-set_property), but manual if you use JS property setters.

Reply:
oh. Ok.